pkg/util/iptables/iptables.go

```diff
@@ -162,6 +163,10 @@ var MinCheckVersion = utilversion.MustParseGeneric("1.4.11")
 // used for port mapping to be fully randomized
 var RandomFullyMinVersion = utilversion.MustParseGeneric("1.6.2")
 
+// RandomFullyMinKernelVersion is the minimum kernel version from which the --random-fully flag is supported,
+// More details please refer to https://github.com/kubernetes/kubernetes/pull/91137
```

Comment:
I actually meant to add a reference for when `--random-fully` was added in the kernel